operator/controllers/csiscaleoperator_controller.go

```diff
@@ -2378,11 +2390,59 @@ func validateMaxUnavailableValue(key string, value string, data map[string]strin
 
 func validateHostNetworkValue(inputSlice []string, key, value string, envMap, invalidEnvValue map[string]string) {
 	logger := csiLog.WithName("validateHostNetworkValue")
-        logger.Info("Validating host network input", "inputHostNetwork", value)
-	
+	logger.Info("Validating host network input", "inputHostNetwork", value)
+
 	if containsStringInSlice(inputSlice, strings.ToUpper(value)) {
-		 envMap[key] = value
-	}else{
+		envMap[key] = value
+	} else {
+		invalidEnvValue[key] = value
+	}
+}
+
+// Input can be either of 0.001 OR 1m
+func validateCPULimitsValue(key string, value string, data map[string]string, invalidEnvValue map[string]string) {
```

Comment:
For both CPU and memory limits validation, it is better to use a library as there can be variety of inputs (e.g. for memory it can be  T, G, M, k or Ei, Pi, Ti, Gi, Mi, Ki), see if we can use resource.ParseQuantity function